Modules/CaPTkSurvival/include/CaPTkSurvival.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CaPTkSurvival_h
+#define CaPTkSurvival_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkCaPTkSurvivalExports.h>
+
+#include <QObject>
+#include <QFuture>
+#include <QFutureWatcher>
+#include <QProgressBar>
```

Comment:
Forward declaration would be fine for progressbar. include it in cpp.

Reply:
This is no longer needed for the time being as the survival predictor does not give progress updates -- I have removed this include.